include/grub/gpt_partition.h

```diff
@@ -186,8 +186,10 @@ struct grub_gpt
   struct grub_gpt_header primary;
   struct grub_gpt_header backup;
 
-  /* Only need one entries table, on disk both copies are identical.  */
-  struct grub_gpt_partentry *entries;
+  /* Only need one entries table, on disk both copies are identical.
+   * The on disk entry size may be larger than our partentry struct so
+   * the table cannot be indexed directly.  */
+  void *entries;
```

Comment:
https://github.com/coreos/grub/pull/35/files#diff-a641efed2b20c406a21e6e537c132a89R433 should similarly be switched to `void *`.